pkg/remote/remote.go

```diff
@@ -54,6 +53,12 @@ func Get(ctx context.Context, ref name.Reference, option types.RemoteOptions) (*
 			continue
 		}
 
+		if option.Platform != nil && option.Platform.Force {
+			if lo.FromPtr(desc.Platform).Satisfies(*option.Platform.Platform) {
+				return nil, xerrors.Errorf("the specified platform not found")
```

Comment:
![image](https://user-images.githubusercontent.com/12510569/234766964-bb57ce16-e461-4522-9f0f-ad63bc2620a1.png)

desc.Platform is nil always
i am not sure if the "satisfies" is as intended. Please check the screenshot. 
Looks like platform is getting filled properly under "desc.platform"
If satisfies is true, why should it return? instead it should have continued with the scan

Reply:
Yes, I didn't expect it. Fixed.